test/e2e/storage/drivers/in_tree.go

```diff
@@ -1283,11 +1283,11 @@ func (v *vSphereDriver) GetDynamicProvisionStorageClass(ctx context.Context, con
 }
 
 func (v *vSphereDriver) PrepareTest(ctx context.Context, f *framework.Framework) *storageframework.PerTestConfig {
-	vspheretest.Bootstrap(f)
 	ginkgo.DeferCleanup(func(ctx context.Context) {
 		// Driver Cleanup function
 		// Logout each vSphere client connection to prevent session leakage
-		nodes := vspheretest.GetReadySchedulableNodeInfos(ctx, f.ClientSet)
+		testContext := vspheretest.NewTestContext(f)
+		nodes := vspheretest.GetReadySchedulableNodeInfos(ctx, testContext, f.ClientSet)
```

Comment:
Since the testContext was created just above, it will have empty `node.VSphere.Client`.